### ISSUE TITLE
1.2.13 crashed on [MMKV clearAll] if MMKV use on +(void)load function

### ISSUE THREAD

**Issue by XiaoWuTongZhi** [Tencent/MMKV#901]
## Note:

An crash issue occur [MMKV clearAll] in multi-process mode .

### The language of MMKV

Objective-C

### The version of MMKV

1.2.13

### The platform of MMKV

iOS

### The installation of MMKV

CocoaPods

### What's the issue?

<img width="1258" alt="image" src="https://user-images.githubusercontent.com/18437833/172591938-0f11fa05-3ed9-405a-81ee-f5c5eb9a39e8.png">

We use MMKV in `+(void)load` func,  then occur crash in multi-process mode, but we found 1.2.13 release note include fix note, so wondering why is this happening?

<img width="1212" alt="v1 2 12" src="https://user-images.githubusercontent.com/18437833/172592570-1a1a78b1-f8fc-41ff-aeac-4941a5b504f2.png">

**Comment by XiaoWuTongZhi** [comment]
Here is MMKV logs:

<img width="637" alt="image" src="https://user-images.githubusercontent.com/18437833/172593911-99875e9b-f40d-4ab8-aad9-7e471bdbd601.png">

**Comment by XiaoWuTongZhi** [comment]
We tried to remove multi-threading, but it still crashed

<img width="986" alt="image" src="https://user-images.githubusercontent.com/18437833/172594306-5dd26253-b269-4965-bc33-e38a5b6aa309.png">

**Comment by lingol** [comment]
No. That release note means subsequential `clearAll()` in another process. 
You can't use an MMKV instance after `close()` in the same process. 
You have to get a new one.

**Comment by lingol** [comment]
And I think we have it documented pretty clearly.
https://github.com/Tencent/MMKV/blob/master/iOS/MMKV/MMKV/MMKV.h#L214

**Comment by XiaoWuTongZhi** [comment]
No, we didn't call the close function, I just created the MMKV instance in +load, and then another MMKV instance will call the clearAll function after theApp launch, and a crash will occur. 
This happens only when MMKV is created in +load case, I don't understand why different MMKV instances affect each other

**Comment by lingol** [comment]
You can't call create an MMKV instance on `+load`. You have to call `+initializeMMKV` inside or after `didFinishLaunch`. Then you may create an MMKV instance.

**Comment by XiaoWuTongZhi** [comment]
But we need to get the MMKV instance on `+load` in some case, what should I do?

**Comment by lingol** [comment]
Don't do it.
